.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,8 @@
+- id: formatr
+  name: formatR
+  description: Uses formatR to format R files
+  entry: scripts/pre-commit-formatR.R
+  language: script
+  types: [text]
+  files: '.*\.[rR]$'
+  require_serial: true
```

Comment:
Why do we need to process the files one by one? This avoids parallel processing when a lot of files are to be processed by the hooks, I don’t think you need that.

Reply:
Pacman fails because it can't lock the library directory, so it forced it to be in parallel. If the installation part is being removed though, this is no longer required.